README.md

```diff
@@ -1,8 +1,11 @@
 # UP
 
-UP is a simple client for testing Prometheus remote-write requests.
-The client writes the specified metric at a chosen interval, where the value of the metric is always the current timestamp in milliseconds.
-It can also read the metric back from a specified endpoint and compare its value against the current time to determine the total write-read latency.
+UP is a simple client for testing Prometheus remote-write and Loki write requests.
+
+For a specified metric the client writes the metric at a chosen interval, where the value of the metric is always the current timestamp in milliseconds. It can also read the metric back from a specified endpoint and compare its value against the current time to determine the total write-read latency.
```

Comment:
Nit: can we keep each new sentence on a newline? It helps edit the document and keeps the diffs legible IMO, since changing one word now will result in a diff for a whole paragraph